apps/web/src/components/ui/news-article-preview-wide.tsx

```diff
@@ -14,47 +17,56 @@ export default function NewsArticlePreviewWide({
 	slug,
 	title,
 }: Readonly<NewsArticlePreviewWideProps>) {
-	const featuredImageSource = urlForImage(featuredImage, 480, 800);
-	const authorImageSource = urlForImage(author.image, 56);
+	const isDesktop = useMediaQuery('(min-width: 48rem)');
+
+	const featuredImageSource = isDesktop
+		? urlForImage(featuredImage, 480, 800)
+		: urlForImage(featuredImage, 267, 350);
+	const authorImageSource = isDesktop
+		? urlForImage(author.image, 56)
+		: urlForImage(author.image, 28);
```

Comment:
@coderabbitai Thanks. I will change it in the upcoming commit.

Reply:
@coderabbitai the `isDesktop` is needed here because of external API. So this is OK here. I don't want to change it in this place.

Do not learn this.